tools/scoring/score/constants.py

```diff
@@ -19,14 +19,15 @@
 class FileTypes(str, Enum):
   """The file which is being scored."""
   PROPOSED = 'proposed'
-  """.The file which is being compared against."""
+  """The file which is being compared against."""
   SOLUTION = 'solution'
 
 
 class DimensionCategories(str, Enum):
-  """Receives `translations` and scores
-  only reporting entities in bulk."""
+  """Dimensions in this category receive `translations`
```

Comment:
can you remove these please?

Reply:
The comments?

Reply:
These won't be in the diff when https://github.com/google/digitalbuildings/pull/519 is merged.